Sources/SPTDataLoaderRequestTaskHandler.m

```diff
@@ -198,7 +209,23 @@ - (BOOL)mayRedirect
 - (void)start
 {
     self.started = YES;
-    self.executionBlock();
+    [self.session.configuration.URLCache getCachedResponseForDataTask:(NSURLSessionDataTask*)self.task
+                                                    completionHandler:^(NSCachedURLResponse *cachedResponse) {
+        NSLog(@"getCachedResponseForDataTask completionHandler got response %p", (__bridge void*)cachedResponse);
+        if (cachedResponse) {
+            [self.task cancel];
```

Comment:
this is dangerous. There is no way to check the etag is still valid without contacting the server, for example

Reply:
From the documentation is not clear if expired content in the cache will be served or discarded